### PR TITLE
Maxstory

### PR DESCRIPTION
**Pull request by iiifilo**
"Please review this, @casey-collab"

### CODE REVIEW

File: espanol/cocinero/cocinero.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Beatito encontro al cocinero con cara de preocupado.
+
+[El cocinero huyo sin dar explicaciones](huida/trabajo.md)
```
Comment (casey-collab) [inline espanol/cocinero/cocinero.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: espanol/huida/trabajo.md [hunk 2]
```diff
@@ -0,0 +1 @@
+El cocinero dijo que no le gusta cocinar para muchas personas.
```
Comment (casey-collab) [inline espanol/huida/trabajo.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: espanol/muralla/abrir/abrir.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 Apresurado Beatito, fue tropezando queriendo llegar lo antes posible con el guardia
 y cuando estaba apunto de llegar comprobó que el enemigo se acercaba a lo lejos.
 
+[Se dirigio al templo y cuando llego no tenia fuerzas para abrir la puerta ](templo/puerta.md)
```
Comment (casey-collab) [inline espanol/muralla/abrir/abrir.md]: This link is also invalid.

File: espanol/templo/puerta.md [hunk 4]
```diff
@@ -0,0 +1 @@
+Los golpes a la puerta interrumpieron al predicador.
```
Comment (casey-collab) [inline espanol/templo/puerta.md]: This file has also been added to a directory that already contained a page.

Comment (casey-collab) [inline espanol/templo/puerta.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

File: espanol/huida/trabajo.md (hunk 2 above)
Comment (casey-collab) [inline espanol/huida/trabajo.md]: This new file also has no link to it.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by iiifilo** [comment]
"Please review this, @casey-collab"

File: espanol/huida/trabajo.md (hunk 2 above)
Comment (casey-collab) [inline espanol/huida/trabajo.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: espanol/templo/puerta.md (hunk 4 above)
Comment (casey-collab) [inline espanol/templo/puerta.md]: This file has also been added to a directory that already contained a page.

Comment (casey-collab) [inline espanol/templo/puerta.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

File: espanol/huida/trabajo.md (hunk 2 above)
Comment (casey-collab) [inline espanol/huida/trabajo.md]: This new file also has no link to it.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by iiifilo** [comment]
"Please review this, @casey-collab"

File: espanol/huida/trabajo.md (hunk 2 above)
Comment (casey-collab) [inline espanol/huida/trabajo.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: espanol/templo/puerta.md (hunk 4 above)
Comment (casey-collab) [inline espanol/templo/puerta.md]: This file has also been added to a directory that already contained a page.

Comment (casey-collab) [inline espanol/templo/puerta.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

File: espanol/huida/trabajo.md (hunk 2 above)
Comment (casey-collab) [inline espanol/huida/trabajo.md]: This new file also has no link to it.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by iiifilo** [comment]
"Please review this, @casey-collab"

File: espanol/huida/trabajo.md (hunk 2 above)
Comment (casey-collab) [inline espanol/huida/trabajo.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

Comment (casey-collab) [inline espanol/huida/trabajo.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by iiifilo** [comment]
"Please review this, @casey-collab"

**Comment by casey-collab** [comment]
Looks great! :thumbsup:

**Pull request merged**